app.js

```diff
@@ -12,4 +12,9 @@ app.get('/api/topics', getTopics);
 app.get('/api/articles', getArticles);
 
 
+app.use((err, req, res, next) => {
```

Comment:
This is currently not required for the tickets up to this point so please remove.